nimbus/src/enrollment.rs

```diff
@@ -536,9 +541,9 @@ impl<'a> EnrollmentsEvolver<'a> {
         let is_user_participating = get_global_user_participation(db, writer)?;
         let experiments_store = db.get_store(StoreId::Experiments);
         let enrollments_store = db.get_store(StoreId::Enrollments);
-        let existing_experiments = experiments_store.collect_all::<Experiment>(writer)?;
-        let existing_enrollments =
-            enrollments_store.collect_all::<ExperimentEnrollment>(&writer)?;
+        let existing_experiments: Vec<Experiment> = experiments_store.collect_all(writer)?;
+        let existing_enrollments: Vec<ExperimentEnrollment> =
+            enrollments_store.collect_all(writer)?;
```

Comment:
These explicit type annotations are to avoid the two-arg-turbofish issue noted above.